### PR TITLE
CI: add integration tests to github action yaml

### PR DESCRIPTION
**Pull request by peter0083**
- reference to #17 
- add steps to run:
  -  Solid Authorization tests
  -  Solid API tests

### CODE REVIEW

**Review by hrchu — CHANGES_REQUESTED** [review]

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -30,4 +30,27 @@ jobs:
       run: pip install pytest
     - name: Run pytest
       run: pytest
+  integration_tests:
+    name: Integration tests 
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: # matrix testing strategy
```
[the remark this answers is not in the document]
Comment (peter0083) [reply .github/workflows/continuous-integration.yml:37]: Noted. Thank you for the review and feedback @hrchu . On which version should we run the integration test? Python 3.9? or Python 3.10?

[the remark this answers is not in the document]
Comment (hrchu) [reply .github/workflows/continuous-integration.yml:37]: anything you say😆

**Review by hrchu — CHANGES_REQUESTED** [review]

**Comment by peter0083** [comment]
thanks @hrchu, then let's do "the latest and hottest Python version minus one": Python 3.9.

**Comment by hrchu** [comment]
> thanks @hrchu, then let's do "the latest and hottest Python version minus one": Python 3.9.

Good idea!

**Pull request merged**